.github/workflows/pr-build.yml

```
@@ -40,4 +40,20 @@ jobs:
       - name: Build with Gradle
         uses: hypertrace/github-actions/gradle@main
         with: 
-          args: dockerBuildImages
\ No newline at end of file
+          args: dockerBuildImages
+
+      - name: Scan docker image
+        uses: azure/container-scan@v0.1
+        with:
+          image-name: hypertrace/java:11
+        env:
+          DOCKLE_HOST: "unix:///var/run/docker.sock"
+        continue-on-error: false
```

should scan both regardless of first error, put the required check in gha rather than the job
```suggestion
        continue-on-error: true
```